### DISCUSSION TITLE
World Map Plugin

### DISCUSSION THREAD

**Discussion by EnuDelrum** [runelite/runelite#17113] [Ideas]
In the World Map plugin, please add the teleportation jewellery icons for the [Ring of the elements](https://oldschool.runescape.wiki/w/Ring_of_the_elements) and [Ring of shadows](https://oldschool.runescape.wiki/w/Ring_of_shadows).

**Comment by YvesW** [comment]
FYI, PRs for this already exist.
https://github.com/runelite/runelite/pull/17100
https://github.com/runelite/runelite/pull/17099
https://github.com/runelite/runelite/pull/17024
